.gitlab-ci.yml

```diff
@@ -114,7 +114,7 @@ test:
     - check-std
   script:
     - for crate in ${ALL_CRATES}; do
-        cargo test --verbose --all-features --release --manifest-path ${crate}/Cargo.toml;
+        cargo test --verbose --all-features --manifest-path ${crate}/Cargo.toml;
```

Comment:
This should not be faster in net result since test runs normally rum much slower in non `--release` than compilation takes.

Reply:
I did some measurements and testing with or without `--release` with caching, do not make much difference for ink.